### ISSUE TITLE
elementary installation

### ISSUE THREAD

**Issue by Bownsie** [elementary/feedback#66]
Just trying to install the altest version on a 2008 macbook. Had an older version (Loki) on my dual boot Windows machine. After wasting a lot of time redoing USB installation media and getting repeated warnigns about not being accessible (advice is to ignore). I tried to test  Demo Mode. It seems that in Demo Mode, elementary doesn't even recognise WiFi, so I can't test linking to Internet (which has stoped working with Loki on my other machine). 

Basically, I'm afriad that elementary OS imay be something for consenting nerds, but is completely useless for most people, even with some computer literacy.

I'll pass and will be removing it comepletely from all my hardware.

I'll be ignoring replies, so don't be surprised or offended.

**Comment by danrabbit** [comment]
Closing since OP said they're going to ignore any replies so there's not anything to do here

**Comment by Bownsie** [comment]
It's just feedback, take it or leave it. Doesn't work for me. Bye and good luck.
